gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Nov 17 13:44:38 EET 2020
+#Tue Jun 04 18:12:06 EEST 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Does it make sense to align it with [Core SDK PR](https://github.com/salemove/android-sdk/pull/717/files) where it was changed to 8.7?

Reply:
Upgraded [3b4f7f9](https://github.com/salemove/android-sdk-widgets/pull/1044/commits/3b4f7f9c236ce3be0660d7eb548b3ea78180b438)